akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -259,6 +261,19 @@ class Unzip[A, B] private (override val shape: FanOutShape2[(A, B), A, B],
   override def named(name: String): Unzip[A, B] = withAttributes(OperationAttributes.name(name))
 }
 
+/**
+ * Takes a stream of pair elements and splits each pair to two output streams.
+ *
+ * '''Emits when''' all of the outputs stops backpressuring and there is an input element available
+ *
+ * '''Backpressures when''' any of the outputs backpressures
+ *
+ * '''Completes when''' upstream completes
+ *
+ * '''Cancels when''' any downstream cancels
+ */
+object UnzipWith extends UnzipWithApply
```

Comment:
Good, also needs to be mirrored in `javadsl`, see here for ZipWith for inspiration: https://github.com/akka/akka/commit/77c8b5f48bef15b10ee5d113fdcc80497abd7e9c#diff-7676430274ea739b1b1b055d7e2a7ed7R34